Source/Core/Common/NandPaths.cpp

```diff
@@ -54,6 +56,21 @@ void ShutdownWiiRoot()
 {
   if (!s_temp_wii_root.empty())
   {
+    u64 tmd_title_id = Movie::GetTitleId();
+    std::string save_path = Common::GetTitleDataPath(tmd_title_id, Common::FROM_SESSION_ROOT);
+    std::string user_save_path =
+        Common::GetTitleDataPath(tmd_title_id, Common::FROM_CONFIGURED_ROOT);
+    if (File::Exists(save_path + "banner.bin"))
+    {
+      // Backup the existing save just in case it's still needed.
+      if (File::Exists(user_save_path + "banner.bin"))
+      {
+        File::CopyDir(user_save_path, user_save_path + "../backup/");
```

Comment:
This was already being done prior to this PR, but I can move it somewhere else if necessary.